apps/vast/README.md

```diff
@@ -39,4 +39,87 @@ Request an intelligence snapshot of the past 50 days.
 
 ```sh
 ./vast-bridge.py --snapshot=50
-```
\ No newline at end of file
+```
+
+## Bridge Features
+
+This section explains the most important features of the `vast-bridge` and the
+CLI toggles to enable them.
+
+### IoC Matching
+
+VAST can match IoCs either live (pro-feature) or retrospectively via the means
+of usual queries.
```

Comment:
But the sentence says `either live or retrospectively via the means of usual queries.` How would you rephrase it? For me it is already quite clear that `usual queries` are used for matching `retrospectively`.

Reply:
> But the sentence says `either live or retrospectively via the means of usual queries`.

Exactly, that's ambiguous 🙂. Because "via the means [..]" can apply to either "live" or "retrospectively". It boils down to operator precedence of the word "or", which is not as well-defined as in a computer language. :neckbeard: 

Here are suggestions to disambiguate:

- "VAST can match IoCs either live via X or retrospectively via Y."
- "VAST can match IoCs either live (via X) or retrospectively (via Y)."
- "VAST can match IoCs either live or retrospectively, via X or Y respectively." (sounds weird though)